plugins/org.jkiss.dbeaver.model.lsm/grammar/SQLStandardParser.g4

```diff
@@ -86,7 +86,8 @@ identifier: (Introducer characterSetSpecification)? actualIdentifier;
 actualIdentifier: (Identifier|DelimitedIdentifier|nonReserved|Quotted);
 
 // data types
-dataType: (datetimeType|intervalType|qualifiedName|anyWordsWithProperty (CHARACTER SET characterSetSpecification)?);
+dataType: (arrayType|datetimeType|intervalType|qualifiedName|anyWordsWithProperty (CHARACTER SET characterSetSpecification)?);
+arrayType: ARRAY LeftParen dataType RightParen; // TODO parse it conditionally on the database engine?
```

Comment:
I agree with this. This is a ClickHouse-specific syntax. Perhaps a debt for the future?